torchok/__main__.py

```diff
@@ -30,10 +31,21 @@ def entrypoint(config: DictConfig):
     schema = OmegaConf.structured(ConfigParams)
     # Merge structure with config
     config = OmegaConf.merge(schema, config)
+    # Seed everything
+    if config.seed_params is not None:
+        seed_everything(**config.seed_params)
     # Create task
     model = torchok.TASKS.get(config.task.name)(config)
     trainer = create_trainer(config)
     if entrypoint == 'train':
+        if config.trainer.auto_lr_find:
+            # Run learning rate finder
+            lr_finder = trainer.tuner.lr_find(model)
+            suggested_lr = lr_finder.suggestion()
+            for i in range(len(config.optimization)):
```

Comment:
PL does not support several optimizers. Throw error if got this more than one optimizers